test/commands/instance/spawn.test.ts

```diff
@@ -2,13 +2,15 @@ import { expect, test } from '@oclif/test'
 
 describe('instance:spawn', () => {
   test
+    .timeout(10000)
     .stdout()
     .command(['instance:spawn'])
     .it('Spawns a new Kuzzle', ctx => {
       expect(ctx.stdout).to.contain('Kuzzle version 2 is launching')
     })
 
   test
+    .timeout(10000)
```

Comment:
shall we add https://eslint.org/docs/rules/no-magic-numbers ?

Reply:
This rule seems very interesting! In that specific case, I doubt it's really worth because IMHO there is no gain between `timeout(TIMEOUT)` and `timeout(10000)` but I think we could use this rule on Kuzzle core. What do you think @scottinet ?

Reply:
I agree, magic numbers are rarely a good thing. 
But this rule must be disabled for unit tests for obvious reasons.

Reply:
Looks like we have a new es lint rule on Kuzzle core :D